react/features/chat/middleware.ts

```diff
@@ -530,8 +526,7 @@ function _handleReceivedMessage({ dispatch, getState }: IStore,
 
     // skip message notifications on join (the messages having timestamp - coming from the history)
     const shouldShowNotification = userSelectedNotifications?.['notify.chatMessages']
-        && !hasRead && !isReaction
-        && (!timestamp || lobbyChat);
+        && !hasRead && !isReaction && !timestamp;
```

Comment:
Did you intentionally drop the lobbyChat here?

Reply:
I don't think so! Maybe that got removed during the rebase... Will have a look!

Reply:
It is fixed now! I found one more place like this which I've also just fixed!